tests/quantization/quanto_integration/test_quanto.py

```diff
@@ -429,3 +431,28 @@ def test_quantize_activation(self):
         with self.assertRaises(ValueError) as e:
             AutoModelForCausalLM.from_pretrained("bigscience/bloom-560m", quantization_config=quantization_config)
         self.assertIn("We don't support quantizing the activations with transformers library", str(e.exception))
+
+
+@require_torch_gpu
+class QuantoKVCacheQuantizationTest(unittest.TestCase):
+    @slow
+    @require_read_token
+    def test_quantized_cache(self):
+        EXPECTED_TEXT_COMPLETION = [
+            "Simply put, the theory of relativity states that 1) time is relative, and 2) space is relative.\nThe theory of relativity is a theory of time and space.\nThe theory of relativity is a theory of time and",
+            "My favorite all time favorite condiment is ketchup. I love it. I love it so much that I have a hard time believing that it was invented in the 17th century. I mean, I know it was invented in",
+        ]
+
+        prompts = [
+            "Simply put, the theory of relativity states that ",
+            "My favorite all time favorite condiment is ketchup.",
+        ]
+        tokenizer = LlamaTokenizer.from_pretrained("meta-llama/Llama-2-7b-hf", pad_token="</s>", padding_side="left")
+        model = LlamaForCausalLM.from_pretrained(
+            "meta-llama/Llama-2-7b-hf", device_map="sequential", torch_dtype=torch.float16
+        )
+        inputs = tokenizer(prompts, return_tensors="pt", padding=True).to(model.device)
```

Comment:
```suggestion
        inputs = tokenizer(prompts, return_tensors="pt", padding=True).to(torch_device)
```
can you import `torch_device` from `testing_utils` ? I think `model.device` might return the meta device if the model is offloaded to CPU or disk